### DISCUSSION TITLE
Παραδοτέο 2 - Π2019166

### DISCUSSION THREAD

**Discussion by olumpiaalexopoulou** [courses-ionio/sw#1258] [Show and tell]
Για την ολοκλήρωση αυτού του παραδοτέου χρησιμοποίησα το ntfy ώστε να στείλω ένα notification στο κινητό μου για τον υπολογισμό της ώρας μέσα από ένα python script που έτρεξα. 
Το αποτέλεσμα φαίνεται στο παρακάτω link :
  - [Asciinema Link](https://asciinema.org/a/YTnQ5e1akGJ1DMFek5sILKho9)
 
[Η Αναφορά μου](https://github.com/olumpiaalexopoulou/sw/tree/2019166/projects/2019166)

**Comment by StavroulaAnatolaki** [comment]
το command prompt πρεπει να είναι ο ΑΜ
